examples/check-examples

```diff
@@ -2,12 +2,13 @@
 
 set -e
 
-for file in $(ls python/*py | xargs -n 1 basename)
+for name in `cat maintained-notebooks.txt`
```

Comment:
Do you think we would have notebooks that are generated with python but that we'd like to skip in the `check-examples` and `gen-nb` scripts? If not, I think getting all the scripts under python/ is better.

Reply:
Yeah, I think we should check all notebooks at least for now. Do you think we should remove ``maintained-notebooks.txt`` all together? If we don't keep unchecked notebooks then I'm not sure what is the point of this file. We can always bring this file back when needed.

Reply:
Yeah, I think removing `maintained-notebooks.txt` is good :)